blackbelt/dependencies.py

```diff
@@ -70,15 +70,18 @@ def check(dep, list_path, licenses_path, dev=False, debug=False):
     dep_name, dep_version = dep
 
     with tempfile.TemporaryDirectory() as tmp_dir:
-        install(dep_name, dep_version, tmp_dir, dev=dev)
+        try:
+            install(dep_name, dep_version, tmp_dir, dev=dev)
+        except subprocess.CalledProcessError:
+            raise click.BadParameter('The npm package could not be installed')
```

Comment:
As above, in this case, I'd do ...`installed: ', stderr)`

Reply:
Okay, passing the error through might be worth.